actions/telegramaction.py

```diff
@@ -26,8 +26,11 @@ def __init__(self, token, receiver, custom_payload=None, template=None):
 
     def perform(self, paste, analyzer_name=None):
         """Send a message via a Telegram bot to a specified user, without checking for errors"""
-        # if self.template:
-        #    text = self.template.format()
-        text = "New paste matched by analyzer '{0}' - Link: {1}".format(analyzer_name, paste.full_url)
+        if self.template is None:
+            self.template = "New paste matched by analyzer '{analyzer_name}' - Link: {url}"
```

Comment:
Please check the opposite state (if `self.template` is **not** None).
If it **is** None, the given string should be used. If it is set, there should be a way to format the **user given** template string as well.

### Example
The template text, created by a user is the following:
```New Paste! Details below:
Name: {name}
Analyzer: {analyzer_name}
...
Body: {paste_body}
Link: {url}
```

That text could be passed to the TelegramAction. When the perform method is called, the existing parameters (all fields from the `paste` class and the `analyzer_name` should be inserted into the message if possible. 

https://stackoverflow.com/questions/19799609/leaving-values-blank-if-not-passed-in-str-format

That could help a lot (check the `format_map` part for Python 3.2 of the top answer).